pyaedt/hfss.py

```diff
@@ -2969,7 +2970,8 @@ def validate_full_design(self, dname=None, outputdir=None, ports=None):
             val_list.extend(temp2_msg)
 
         # Run design validation and write out the lines to the log.
-        temp_val_file = os.path.join(os.environ["TEMP"], "\\val_temp.log")
+        temp_fld = tempfile.gettempdir()
+        temp_val_file = os.path.join(temp_fld, "val_temp.log")
```

Comment:
Nitpick: let's not use folder anymore and instead directory.  Folder is a (Windows) GUI concept. 

```suggestion
        temp_dir = tempfile.gettempdir()
        temp_val_file = os.path.join(temp_dir, "val_temp.log")
```